### ISSUE TITLE
Ability to automatically install fonts.

### ISSUE THREAD

**Issue by AqilCont** [files-community/files-uwp#1019]
**Is your feature request related to a problem? Please describe.**
Yeah, the original file explorer supported installing fonts through simple selection, right click, and pressing "Install" (or "Install all" for multiple fonts). With Files UWP, you have to individually open the font files to install them, using an in-build Windows tool.

**Describe the solution you'd like**
I would prefer a built-in, integrated way to install fonts for convenience. 
Basically, just another menu item that says "Install" when font file types are selected :D 
![image](https://user-images.githubusercontent.com/32044067/84446720-2519e980-ac14-11ea-8e0d-397997896d60.png)

**Describe alternatives you've considered**
~~switching back to File Explorer~~

**Comment by yaichenbaum** [comment]
@AqilCont Thank you for your feedback, the option to install fonts will be important to get on par with File Explorer functionality.

**Comment by yaichenbaum** [comment]
Merging with #174.

**Comment by yaichenbaum** [comment]
This has been implemented in #1754.
